e2e/tests/trustless_relays.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Trustless Relays
+
+  Scenario: User Wants Help Using The Servicer Command 
```

Comment:
Do we need this scenario?

Reply:
Not sure, so removed it. It will resurface if we need it in the future.